bin/strata-client/src/errors.rs

```diff
@@ -43,13 +39,7 @@ pub enum ConfigError {
     #[error("can't traverse into primitive: {0}")]
```

Comment:
But regarding other comment, it would make sense for the error *message* to be more correct here, it's plausible someone might think a path like `foo.7.bar` is valid for trying to traverse into index 7 of a toml list.

Reply:
maybe `tried to traverse into non-table key: {0}`?